src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-detail/tutorial-group-detail.component.html

```diff
@@ -121,9 +121,12 @@ <h3>{{ 'artemisApp.pages.tutorialGroupDetail.title' | artemisTranslate }}:</h3>
             <span>{{ 'artemisApp.entities.tutorialGroup.language' | artemisTranslate }}</span>
         </dt>
         <dd>
-            <span *ngIf="tutorialGroup.language">
-                {{ (tutorialGroup.language === GERMAN ? 'artemisApp.generic.german' : 'artemisApp.generic.english') | artemisTranslate }}
-            </span>
+            <ng-container *ngIf="tutorialGroup.language; else noLanguage">
+                <span>{{ tutorialGroup.language }}</span>
```

Comment:
I like the ternary operator from the occurrence below more.
You can also think about using - as placeholder as some other tables do

Reply:
Replaced with ternary operator in commit: f0e3d6e3f69d67df42e50507c116ff73e0619916